proto/v1/property_values.proto

```diff
@@ -37,11 +37,13 @@ message InPropertyValuesRequest {
   // [Optional]
   // The pagination token for getting the next set of entries. This is empty
   // for the first request and needs to be set in the subsequent request.
-  // This is the value returned from a prior call to InPropertyValuesResponse
+  // This is the value returned from a prior call to PropertyValuesResponse
   string next_token = 4;
+  // Direction can only be "in" and "out"
+  string direction = 5;
```

Comment:
Should we use enum (preferable), or boolean?

Reply:
This field is parsed by the ESP from the REST url where we could only get a string.